code/game/objects/structures/gym/punching_bag.dm

```diff
@@ -0,0 +1,69 @@
+/obj/structure/punching_bag
+	name = "punching bag"
+	desc = "A punching bag. Can you get to speed level 4???"
+	icon = 'icons/obj/gym_equipment.dmi'
+	icon_state = "punchingbag"
+	anchored = TRUE
+	layer = ABOVE_MOB_LAYER
+	///List of sounds that can be played when punched.
+	var/static/list/hit_sounds = list(
+		'sound/weapons/genhit1.ogg',
+		'sound/weapons/genhit2.ogg',
+		'sound/weapons/genhit3.ogg',
+		'sound/weapons/punch1.ogg',
+		'sound/weapons/punch2.ogg',
+		'sound/weapons/punch3.ogg',
+		'sound/weapons/punch4.ogg',
+	)
+
+/obj/structure/punching_bag/Initialize(mapload)
+	. = ..()
+
+	AddElement( \
+		/datum/element/contextual_screentip_bare_hands, \
+		lmb_text = "Punch", \
+	)
```

Comment:
does this really have to be multilined? if it needs it it's probably fine but it really doesn't appear to need it at-a-glance

Reply:
this is copy pasted from when i split the .dm files.